library/Vanilla/Embeds/TwitchEmbed.php

```diff
@@ -0,0 +1,198 @@
+<?php
+/**
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license GPL-2.0
```

Comment:
This type of license annotation is not actually valid. It should be

```
@license https://opensource.org/licenses/GPL-2.0 GPL-2.0
```

Update your PHPStorm file comment config.